[2025-09-05T09:28:02.327+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-05T09:28:02.351+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T09:27:00+00:00 [queued]>
[2025-09-05T09:28:02.359+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T09:27:00+00:00 [queued]>
[2025-09-05T09:28:02.359+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-05T09:28:02.373+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-05 09:27:00+00:00
[2025-09-05T09:28:02.383+0000] {standard_task_runner.py:63} INFO - Started process 3180453 to run task
[2025-09-05T09:28:02.386+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-05T09:27:00+00:00', '--job-id', '113147', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmppuef1hyz']
[2025-09-05T09:28:02.389+0000] {standard_task_runner.py:91} INFO - Job 113147: Subtask save_foreign_1
[2025-09-05T09:28:02.445+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T09:27:00+00:00 [running]> on host 1877907bc043
[2025-09-05T09:28:02.538+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T09:27:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-05T09:27:00+00:00'
[2025-09-05T09:28:02.539+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-05T09:28:02.553+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T09:28:02.887+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-05T09:28:02.889+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T09:28:03.282+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-05T09:28:03.284+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T09:28:03.645+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-05T09:28:03.646+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T09:28:03.990+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-05T09:28:03.991+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-05T09:28:03.992+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-05T09:28:04.010+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-05T09:27:00+00:00, execution_date=20250905T092700, start_date=20250905T092802, end_date=20250905T092804
[2025-09-05T09:28:04.084+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-05T09:28:04.127+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-05T09:28:04.129+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
